sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/Expression.java

```diff
@@ -29,5 +29,5 @@
   /**
    * Format the expression as a human readable SQL-like string.
    */
-  String describe();
+  default String describe() { return this.toString(); }
```

Comment:
I realized that almost all the implementations override both `toString` and `describe`, so making this change to make it simpler.